tests/unit/test_parameters/test_parameter_sets_class.py

```diff
@@ -1,23 +1,22 @@
 #
 # Tests for the ParameterSets class
 #
-from tests import TestCase
-
+import pytest
+import re
 import pybamm
-import unittest
 
 
-class TestParameterSets(TestCase):
+class TestParameterSets:
     def test_name_interface(self):
         """Test that pybamm.parameters_sets.<ParameterSetName> returns
         the name of the parameter set and a depreciation warning
         """
-        with self.assertWarns(DeprecationWarning):
+        with pytest.warns(DeprecationWarning):
             out = pybamm.parameter_sets.Marquis2019
-            self.assertEqual(out, "Marquis2019")
+            assert out == "Marquis2019"
 
         # Expect error for parameter set's that aren't real
```

Comment:
```suggestion
        # Expect an error for parameter sets that aren't real
```

Feel free to make any grammar corrections that you find while going through the codebase!